eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -196,40 +196,50 @@ public void startup(EwonConnectorSettings settings) {
       // Verify integrity of tag history storage information if tag history enabled
       if (historyEnabled && StringUtils.isBlank(tagHistoryStore)) {
          logger.warn(
-               "History sync is enable, but no history provider has been specified for storage. No data will be stored.");
+             "History synchronization is enabled, but no history provider " +
+                 "has been specified for storage. No data will be stored.");
       }
 
       // Output warning if tag name check is disabled
       if (tagNameCheckDisabled) {
-         logger.warn("Tag name checking is disabled. Tags with unsupported characters may not " +
-             "work properly.");
+         logger.warn("Strict checking for allowed tag name characters is " +
+                 "disabled. Tags with unsupported characters may not work properly");
       }
 
       // Load and store synchronization data, and create it if necessary
       syncData = gatewayContext.getPersistenceInterface().find(EwonSyncData.META, 1L);
       if (syncData == null) {
-         logger.info("Ewon sync data not found, initializing.");
+         logger.info("Existing Ewon synchronization data not found, " +
+                 "creating new data store");
          syncData = gatewayContext.getLocalPersistenceInterface().createNew(EwonSyncData.META);
          syncData.setLong(EwonSyncData.Id, 1L);
          gatewayContext.getLocalPersistenceInterface().save(syncData);
       }
 
       // Load and register polling interval configuration information
       long pollRateMS = TimeUnits.toMillis(settings.getPollRate().doubleValue(), TimeUnits.MIN);
-      logger.debug("Configuring polling for {} ms", pollRateMS);
       if (pollRateMS > 0) {
+         logger.debug("Configuring polling for {} ms", pollRateMS);
          gatewayContext.getExecutionManager().register("ewon", "syncpoll", this::run,
                (int) pollRateMS);
       }
+      else if (pollRateMS < 0) {
+         logger.warn("Cannot configure polling for {} ms. " +
+                 "Try an interval greater than 0 mins", pollRateMS);
+      }
 
       // Load and register realtime polling interval configuration information
       long livePollRateMS =
               TimeUnits.toMillis(settings.getLivePollRate().doubleValue(), TimeUnits.SEC);
-      logger.debug("Configuring live polling for {} ms", livePollRateMS);
       if (pollRateMS > 0) {
+         logger.debug("Configuring live polling for {} ms", livePollRateMS);
          gatewayContext.getExecutionManager().register("ewon", "synclive", this::runLive,
                  (int) livePollRateMS);
       }
+      else if (livePollRateMS < 0) {
+         logger.warn("Cannot configure realtime polling for {} ms. " +
```

Comment:
Would it be more friendly to the user to output the poll rate in seconds rather than milliseconds?